src/cloud-code/Parse.Cloud.js

```diff
@@ -84,14 +85,16 @@ ParseCloud.job = function (functionName, handler) {
  * @name Parse.Cloud.beforeSave
  * @param {(String|Parse.Object)} arg1 The Parse.Object subclass to register the after save function for. This can instead be a String that is the className of the subclass.
  * @param {Function} func The function to run before a save. This function can be async and should take one parameter a {@link Parse.Cloud.TriggerRequest};
+ * @param {(Object|Function)} validator An optional function or object to help validating cloud code. This object should be a {@link Parse.Cloud.ValidatorObject}, and this function can be an async function and should take one parameter a {@link Parse.Cloud.ValidatorFunction}. Returning false from this function will return a "validation fail" error.
```

Comment:
`Returning false from this function will return a "validation fail" error.` Can be removed.